src/components/AddTodoForm/AddTodoForm.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext, useState } from 'react';
+import { TodosContext } from '../TodosProvider';
+
+export const AddTodoForm: React.FC = () => {
+  const {
+    handleFormSubmit,
```

Comment:
it is overkill to store functions in context

Reply:
why ? the contest helps me handle a bunch of props ...